compiler/rustc_feature/src/unstable.rs

```diff
@@ -390,6 +390,8 @@ declare_features! (
     /// Allows `async` trait bound modifier.
     (unstable, async_trait_bounds, "1.85.0", Some(62290)),
     /// Allows using C-variadics.
+    (unstable, avx10_target_feature, "CURRENT_RUSTC_VERSION", Some(138843)),
+    /// Allows using Intel AVX10 target features and intrinsics
     (unstable, c_variadic, "1.34.0", Some(44930)),
```

Comment:
The doc comments are swapped.

```suggestion
    /// Allows using Intel AVX10 target features and intrinsics.
    (unstable, avx10_target_feature, "CURRENT_RUSTC_VERSION", Some(138843)),
    /// Allows using C-variadics.
    (unstable, c_variadic, "1.34.0", Some(44930)),
```

Reply:
thanks for pointing it out